lib/modules/datasource/custom/index.ts

```diff
@@ -64,4 +68,24 @@ export class CustomDatasource extends Datasource {
       return null;
     }
   }
+
+  private async fetchPlainFormat(url: string): Promise<unknown> {
+    const response = await this.http.get(url, {
+      headers: {
+        'Content-Type': 'text/plain',
```

Comment:
```suggestion
        'Accept': 'text/plain',
```
The client sends the `Accept` header to get the same back as `Content-Type`